### ISSUE TITLE
Link

### ISSUE THREAD

**Issue by jamesmelzer** [JDRF/designsystem#86]
### Deliver

* Visual style for links in text
* Considerations: underline, hover state, color, visited color, weight

### Process

Build 
Doc 
Pub

**Comment by patrickmckowen** [comment]
For consideration, use a left to right animation on hover for the link underline: https://codepen.io/mxttsco/pen/KgodYm

**Comment by jamesmelzer** [comment]
In terms of use cases, is this all the time for links, or just certain high-profile ones?

**Comment by patrickmckowen** [comment]
I can see it being used in long form text.  Ismael is currently exploring the use of Text Buttons vs plain links and how they are used in components and may differ.

**Comment by jamesmelzer** [comment]
Looks good in Links, Long Form Text, and Button art boards. Okay to merge #86 PR (which is really button).

**Comment by jdrf-digitaldesigner** [comment]
@patrickmckowen Making sure that you're okay with the updates before closing. Thx

**Comment by patrickmckowen** [comment]
@jdrf-digitaldesigner looks good.

For posterity, I'll call out that we are _not_ changing the link style for visited links.  I actually couldn't find any examples of websites doing this.  I checked media sites like Vox and NY Times, big players that put a focus on accessibility like Microsoft and Google, design leaders like Intercom and Stripe.  The only example I could find, NN Group's site, change their link color so subtly that you can't even tell without squinting.

**Comment by jdrf-digitaldesigner** [comment]
@patrickmckowen Makes sense. I'll update the default style to "default/visited" to prevent any confusion. Thx
